sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Azure.Iot.ModelsRepository.Samples
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            // Forward all the events written to the console output with a specific format.
+            using AzureEventSourceListener listener = new AzureEventSourceListener(
+                (e, message) =>
+                    Console.WriteLine("[{0:HH:mm:ss:fff}][{1}] {2}", DateTimeOffset.Now, e.Level, message),
+                level: EventLevel.Verbose);
+
+            await ResolveExistingAsync();
+            await TryResolveButNotFoundAsync();
+        }
+
+        private static async Task ResolveExistingAsync()
+        {
+            var dtmi = "dtmi:com:example:TemperatureController;1";
```

Comment:
Why aren't we using snippets?

Reply:
We will, we have a task to add readme.md file for samples